assets/notebooks/assembly_report.qmd

```diff
@@ -164,14 +164,82 @@ We can glean a good amount of information about the sequenced species from the k
 #### Ploidy
 Does it show the correct ploidy?
 
-::: {.callout-note title="Interpretation" collapse="true"}
-A Kmer histogram counts the frequency of k-mers in the data. Some parts of the genome are unique, and
-there are expected to be *coverage* times copies of these k-mers in the data. If the organism is diploid,
-the regions unique to a haplotype should have *coverage* times copies of these k-mers, and regions shared
-between both haplotypes should have *coverage* times two copies of these k-mers. This leads to formations
-of peaks in the k-mer histogram. A haploid organism should have one peak, while a diploid should have two peaks.
+
+::: {.callout-note title="what is GenomeScope and how to interpret the GenomeScope plot" collapse="true"}
+#### **GenomeScope Overview**
+
+**GenomeScope** is a computational tool designed to analyze raw sequencing data and provide insights into the genome's characteristics, such as its **size**, **heterozygosity rate**, and **repeat content**. It works by analyzing **k-mer frequency distributions** derived from sequencing reads and fitting a mathematical model to these distributions.
+
+---
+
+##### **How GenomeScope Works**
+
+1. **Input Data**  
+   GenomeScope uses a **k-mer frequency histogram** generated by tools like **Jellyfish**.  
+   - A **k-mer** is a sequence of *k* nucleotides.  
+   - The histogram represents how often each unique k-mer appears in the sequencing data.
+
+2. **Model Fitting**  
+   GenomeScope fits a **mixture model of negative binomial distributions** to the k-mer histogram. This model accounts for:  
+   - **Unique sequences**  
+   - **Repeated sequences**  
+   - **Heterozygous and homozygous regions**
+
+3. **Output**  
+   The tool estimates:  
+   - **Genome size**  
+   - **Heterozygosity**  
+   - **Repeat content**  
+   It also generates **plots** and **tables** summarizing these properties.
+
+---
+
+##### **Interpreting the GenomeScope Plot**
+
+The plot typically includes:
+
+- **X-axis**: Coverage (how many times each k-mer appears in the data).  
+- **Y-axis**: Frequency (number of k-mers at each coverage level).  
+
+###### **Peaks**  
+- The **first peak** (lower coverage) corresponds to **heterozygous regions**.  
```

Comment:
This needs to be prefixed with the assumption, we're looking at a diploid organism.